welcome.html

```diff
@@ -14,6 +16,10 @@
           <P class="version-tag">v1.0.0</P>
         </div>
       </div>
+      <input class="click-icon" type="checkbox" id="click-icon">
+      <label for="click-icon" class="menu-btn">
+        <i class="fas fa-bars burger-icon"></i>
```

Comment:
it is the link of three line icon or hamburger icon. and "fas  fa-bars" is the name of icon Since I take the icon from google 

![Uploading Screenshot 2024-07-10 at 0.47.38 PM.png…]()